core/pkg/adapters/grpc/websocket_grpc.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2021 Nitric Pty Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package grpc
+
+import (
+	"context"
+
+	"google.golang.org/grpc/codes"
+
+	pb "github.com/nitrictech/nitric/core/pkg/api/nitric/websocket/v1"
+	"github.com/nitrictech/nitric/core/pkg/plugins/websocket"
+)
+
+// GRPC Interface for registered Nitric Storage Plugins
```

Comment:
Old comment needs to be replaced/removed

Reply:
Done